environments/production.env

```diff
@@ -8,6 +8,7 @@ OPENID_VERIFY_URL=https://build.snapcraft.io/login/verify
 LP_API_URL=https://api.launchpad.net
 STORE_API_URL=https://dashboard.snapcraft.io/dev/api
 STORE_DEVPORTAL_URL=https://dashboard.snapcraft.io/dev
+STORE_CPI_API_URL=https://search.apps.ubuntu.com/api/v1
```

Comment:
Let's call this `STORE_SEARCH_API_URL` or something.  We sometimes refer to it as CPI but that's actually an abbreviation for the previous implementation (click-package-index), which has now been replaced by a new implementation, so it's even more opaque than your typical initialism ...